build.gradle

```diff
@@ -72,14 +102,108 @@ animalsniffer {
     annotation = 'io.reactivex.internal.util.SuppressAnimalSniffer'
 }
 
-// support for snapshot/final releases with the various branches RxJava uses
-nebulaRelease {
-    addReleaseBranchPattern(/\d+\.\d+\.\d+/)
-    addReleaseBranchPattern('HEAD')
+task sourcesJar(type: Jar, dependsOn: classes) {
+    classifier = 'sources'
+    from sourceSets.main.allSource
+}
+
+task javadocJar(type: Jar, dependsOn: javadoc) {
+    classifier = 'javadoc'
+    from javadoc.destinationDir
+}
+
+artifacts {
+    archives jar
+    archives sourcesJar
+    archives javadocJar
+}
+
+jar {
+    manifest {
+        name = 'rxjava'
+        instruction 'Bundle-Vendor', 'RxJava Contributors'
+        instruction 'Bundle-DocURL', 'https://github.com/ReactiveX/RxJava'
+        instruction 'Import-Package', '!org.junit,!junit.framework,!org.mockito.*,*'
```

Comment:
What about testNG? 

As an alternative, you can explicitly declare only `org.reactivestreams;version="[1.0,2)"`.
Or we can figure out how to dynamically collect `compile` dependencies to put them here

Reply:
I think these are needed by the OSGi plugin MANIFEST.MF. I don't know how relevant this is.